docs/io.cozy.coachco2.md

```diff
@@ -0,0 +1,19 @@
+# Cozy CoachCO2 settings doctype
+
+This doctype store information about CoachCO2 application settings. There is only one document.
+
+The doctype stores the following data:
+- `account`: {Object} - Relative account, copy of `io.cozy.account` document
+- `bikeGoal`: {Object} - Informations about the bike goal feature
+  - `showAlert`: {boolean} - Either an alert is displayed requesting activation of the bike goal feature
+  - `showAlertSuccess`: {boolean} - Either an alert is displayed when the goal is reached
+  - `activated`: {boolean} - Either the feature is enabled
+  - `sendToDACC`: {boolean} - Either the user has agreed to share their data with the DACC
+  - `onboarded`: {boolean} - Either the user as completed the onboarding
```

Comment:
I don't think "either" is grammatically correct here, it's more "If" or "Whether" :)

Reply:
Whether is fine, thanks